### PR TITLE
fix: use pytds driver in SQL Server sample

### PR DESCRIPTION
**Pull request by shubha-rajan**
## Description

Update sample to use pytds, a pure-python driver instead of pyodbc, so that it can work on Cloud Functions

## Checklist
- [x] I have followed [Sample Guidelines from AUTHORING_GUIDE.MD](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/master/AUTHORING_GUIDE.md)
- [x] README is updated to include [all relevant information](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/master/AUTHORING_GUIDE.md#readme-file)
- [ ] **Tests** pass:   `nox -s py-3.6` (see [Test Environment Setup](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/master/AUTHORING_GUIDE.md#test-environment-setup))
- [ ] **Lint** pass:   `nox -s lint` (see [Test Environment Setup](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/master/AUTHORING_GUIDE.md#test-environment-setup))
- [ ] These samples need a new **API enabled** in testing projects to pass (let us know which ones)
- [ ] These samples need a new/updated **env vars** in testing projects set to pass (let us know which ones)
- [x] Please **merge** this PR for me once it is approved.
- [ ] This sample adds a new sample directory, and I updated the [CODEOWNERS file](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/master/.github/CODEOWNERS) with the codeowners for this sample

### CODE REVIEW

**Review by kurtisvg — APPROVED** [review]

File: cloud-sql/sql-server/sqlalchemy/requirements.txt [hunk 1]
```diff
@@ -1,4 +1,5 @@
 Flask==1.1.2
 SQLAlchemy==1.3.22
-pyodbc==4.0.30
+python-tds==1.10.0
```
Comment (engelke) [inline cloud-sql/sql-server/sqlalchemy/requirements.txt:3]: connection_test is still using pyodbc.

**Review by engelke — APPROVED** [review]
LGTM once tests pass.

**Comment by shubha-rajan** [comment]
The tests are passing locally while running the proxy, but not on Kokoro. Getting this error.
```
sqlalchemy.exc.DBAPIError: (pytds.tds_base.Error) Client does not have encryption enabled but it is required by server, enable encryption and try connecting again
(Background on this error at: http://sqlalche.me/e/13/dbapi)
```
Any ideas why this error would show up here but not locally?

**Comment by kurtisvg** [comment]
> The tests are passing locally while running the proxy, but not on Kokoro. Getting this error.
> 
> ```
> sqlalchemy.exc.DBAPIError: (pytds.tds_base.Error) Client does not have encryption enabled but it is required by server, enable encryption and try connecting again
> (Background on this error at: http://sqlalche.me/e/13/dbapi)
> ```
> 
> I'm in the process of debugging it. Any ideas why this error would show up here but not locally?

Are we using the same instance? Is it possible there is a "required SSL" bit that was flipped for overground?

**Comment by shubha-rajan** [comment]
> Are we using the same instance? Is it possible there is a "required SSL" bit that was flipped for overground?
Different instance, but both are under the Google org so overground rules should be the same. Running the application works locally too, so I don't think the tests passing is a fluke. 

Also, shouldn't using the proxy mean the connection to the server is SSL-encrypted? Is the issue that it expects encryption while connecting to the proxy server on the client side too?

**Pull request merged**